cmd/create/accountroles/cmd.go

```diff
@@ -256,6 +256,10 @@ func run(cmd *cobra.Command, argv []string) {
 		r.Reporter.Errorf("Expected a valid role prefix matching %s", aws.RoleNameRE.String())
 		os.Exit(1)
 	}
+	if !args.hostedCP && strings.HasSuffix(prefix, "-HCP") {
+		r.Reporter.Errorf("'-HCP' suffix is not support for classic account roles")
```

Comment:
I'd say something along the lines of "'-HCP' suffix is reserved/restricted to Hosted Control Planes managed policies" would be clearer for customer

Reply:
Good point changed.

Why would we like to run this query?

Reply:
If someone is using it already it might be confusing for the customer when we start doing this